web_monitoring_diff/tests/test_html_diff.py

```diff
@@ -54,7 +54,6 @@ def inner(**kwargs):
         filename = func.__name__ + '-' + '-'.join(kwargs.values()) + '.html'
         with open(OUTPUT_DIR / Path(filename), 'w') as file:
             file.write(d)
-        return d
```

Comment:
Pytest now logs warnings if you return values from a test (since you might be migrating from a testing framework that asserts something about returned values). We don’t actually need this return value for anything, so it’s safe to just drop the return value here.